mobile-tester.py

```diff
@@ -76,6 +79,11 @@ def run(api_key, api_secret, name, silent):
             "plugins": {
                 "mobileTest": {
                     "enabled": True,
+                    "devices": [
+                        {
+                            "arn": device_arn_1
```

Comment:
@aryeguy   - Instead of mapping directly to AWS ARN, should we map this to our internal store and then map that to AWS. This way our system will be abstracted from the naming conventions of a particular platform as we might switch this to say google cloud tomorrow.

Another option is let user create a test using device and os combo in our project create. Instead of ARN, we pass in ```devices : [ {type: ios, version:10.3 , device: 'iphone 6'} ``` as that comes off as more natural from a user perspective. Thoughts ?